docs/system-admin-guide/integrations/nextcloud/README.md

```diff
@@ -1,79 +1,96 @@
+Once you have copied the values, click on the **Save** button to proceed to the next step in Nextcloud.
 
-![OpenProject_API_key](OpenProject_API_key.png)
+**4. Generate and copy the Nextcloud OAuth values** - in Nextcloud and OpenProject
 
-![OpenProject_API_key_copy](OpenProject_API_key_copy.png)
+As in the previous step, the *OAuth values* are generated automatically, but this time on the Nextcloud end. These values are needed to allow the connection from Nextcloud to OpenProject, so you will need to once again copy them here and paste them in OpenProject (without closing the Nextcloud tab). If you had not clicked the **Done. Continue setup** button in the previous step, you can do so now to proceed to the screen where you will be able to paste the Nextcloud OAuth values in OpenProject.
 
-**Display of OpenProject in the Nextcloud dashboard**
+![Nextcloud_NC_OAuth_values](3_2_04-NC_Step_3.png)
 
-On the Nextcloud dashboard you can add an OpenProject widget. Display the latest changes to your project's work packages to keep an eye on your ongoing project activities directly from your Nextcloud instance.
+![OpenProject_NC_OAuth_values](3_3_01-OP_OAuth_application_details.png)
 
-![Add_OpenProject_widget](Add_OpenProject_widget.png)
+Once these values are entered, you can click the button **Save and complete setup** in your OpenProject tab and the **Yes, I have copied these values** in Nextcloud. Once this is done, your instance configuration is complete.
 
-![Nextcloud_dashboard](Nextcloud_dashboard.png)
+![OpenProject_success](3_4_00 - OAuth - OP Success.png)
 
-In your personal settings in Connected accounts, please remember to also activate the Enable navigation link to display a link to your OpenProject instance in the header navigation.
+![Nextcloud_success](3_2_05-NC_Success.png)
 
-![Nextcloud_connected_account](Nextcloud_connected_account.png)
+**5. Enable the file storages module and activate it in the desired project** - in OpenProject
 
-The link will show here:
+Now that the integration setup is complete and ready to use, there is just one more step for you as administrator: you need to activate the *File storages* module and specify for which projects you would like to enable your new Nextcloud integration.
 
-![Navigation_link_OpenProject](Navigation_link_OpenProject.png)
+To activate the module, go to the **Project settings** in the desired project and access the **Modules** entry in the side menu. There, activate the *File storages* module. Once this is done, a new menu entry called **File storages** appears at the bottom of the side menu, where you can select your new Nextcloud integration.
 
-By activating "enable unified search for tickets" in your personal settings, the Nextcloud dashboard will include OpenProject information in the the built-in universal search:
+![module_activation](Settings_modules.png)
 
-![Unified_search](Unified_search.png)
+![settings_files_storages](Settings_files_storages.png)
```

Comment:
@marcalcobe Here we could link to the user guide, in the sense of "All further steps are explained in the User guide"

Reply:
Hi @wielinde, @psatyal added this information in his last commit.